ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/PullPhysicalPlan.java

```diff
@@ -172,8 +172,7 @@ public QueryId getQueryId() {
   public enum PullPhysicalPlanType {
     // Could be one or more keys
     KEY_LOOKUP,
-    TABLE_SCAN,
-    UNKNOWN
```

Comment:
In any case where we actually have a plan, we always know the kind of plan it is.